charts/render.py

```diff
@@ -114,25 +120,45 @@ def keep_doc(doc):
 
 
 def main():
-    (name, namespace, values, path) = sys.argv[1:]
+    parser = argparse.ArgumentParser()
+    parser.add_argument('name', help="Denotes the name of the chart")
+    parser.add_argument('namespace', help="Metalk8s namespace to deploy unto")
+    parser.add_argument('values', help="Our custom chart values")
+    parser.add_argument(
+        '--service-config',
+        action='append',
+        nargs=2,
+        dest="service_config",
+        help="Ex: --service-config service_name service_configmap_name"
+    )
+    parser.add_argument('path')
+    args, extra = parser.parse_known_args()
```

Comment:
Not sure you want to allow (and ignore) extra arguments. Just using `args = parser.parse_args()` will fail if unknown arguments are provided.

Reply:
Well, we redirect the output to a file using something like `> salt/metalk8s/addons/prometheus-operator/deployed/chart.sls`

We actually want to omit this extra argument

Reply:
The redirect is catch by your shell directly and will **not** be part of the arguments python will get

Reply:
ok